.github/workflows/deps_eager.yml

```diff
@@ -34,8 +40,14 @@ jobs:
         pip install -U --upgrade-strategy eager -r .github/workflows/requirements_eager.txt
 
     - name: Run tests on updated packages
-      run: |
-        pytest -rs --cov=./optimade/
+      run: pytest -rs -s --cov=./optimade/ --cov-report=xml
+      env:
+        OPTIMADE_CI_FORCE_MONGO: 1
+
+    - name: Run tests relevant for index meta-db (using `mongomock`)
+      run: pytest -rs -s --cov=./optimade/ --cov-report=xml --cov-append tests/server/test_middleware.py tests/server/test_server_validation.py tests/server/test_config.py
```

Comment:
```suggestion
      run: pytest -rs --cov=./optimade/ --cov-report=xml --cov-append tests/server/test_middleware.py tests/server/test_server_validation.py tests/server/test_config.py
```

Reply:
Just make a commit --amend to your previous commit/fix

Reply:
Yeah I know, lazily did this one through the web interface on my phone ><